docs/spec-files/message_structure.md

```diff
@@ -75,37 +75,36 @@ The following example shows common elements of a plaintext message. Further deta
 
 However, some attributes are common to many different message types. When metadata about a message means the same thing regardless of context, and when it is susceptible to generic rather than message-specific handling, that metadata can be placed in **headers**. Headers are siblings of `body` and may be added to any message type. They are encrypted and decrypted along with `body` and therefore have an identical audience.
 
-Headers in DIDComm are intended to be extensible in much the same way that headers in HTTP or SMTP are extensible. A few headers are predefined:
+Headers in [DIDComm Messaging](.) are intended to be extensible in much the same way that headers in HTTP or SMTP are extensible. A few headers are predefined:
 
 - **id** - REQUIRED. Message ID. The `id` attribute value MUST be unique to the sender.
 
-- **type** - REQUIRED. Plaintext message type, useful for message handling in application-level protocols. The `type` attribute value MUST be a valid [Message Type URI](protocols.md#message-type-uri), that when resolved gives human readable information about the message. The attribute's value predicts the content in the `body` of the message, for example the presence of other attributes and how they should be processed.
+- **type** - REQUIRED. Plaintext message type, useful for message handling in application-level protocols. The `type` attribute value MUST be a valid [Message Type URI](protocols.md#message-type-uri), that when resolved gives human readable information about the message. The attribute's value SHOULD predict the content in the `body` of the message.
 
-- **typ** - OPTIONAL. Media type of the JWM content. MUST be set to `application/didcomm-plain+json` for **DIDComm plaintext messages**. This tells generic JOSE libraries that the JWM in question is DIDComm rather than a different JWM type and is therefore helpful if messages will be processed in a system that supports a variety of JWMs.
+- **typ** - OPTIONAL. Media type of the JWM content.
 
-    The following table provides an overview of examples for using the media type properties in various DIDComm messages:
+    The following table provides an overview of using the media type properties in supported [DIDComm Messaging](.) messages:
 
     | DIDComm message | `typ` |
     |-----------------|-------|
-    | Encrypted | `application/didcomm-encrypted+json`|
-    | Signed, then encrypted | `application/didcomm-encrypted+json`|
-    | Encrypted, then encrypted | `application/didcomm-encrypted+json` |
+    | Authcrypted and/or anoncrypted | `application/didcomm-encrypted+json`|
+    | Signed and anoncrypted | `application/didcomm-encrypted+json`|
     | Signed | `application/didcomm-signed+json` |
     | Plaintext | `application/didcomm-plain+json`|
```

Comment:
@kdenhartog could you pls check if you are ok with those changes. Idea is to make the table more readable (complete).